examples/kms/main.tf

```diff
@@ -19,6 +19,8 @@ module "domino_eks" {
     enabled = true
   }
   kms = {
-    enabled = true
+    enabled = false
```

Comment:
Couldn't we just lifecycle ignore this so we can have the default we want? Seems really unfortunate to debut with a default other than our preference due to pre-release usage.

Reply:
Given that this is a very intrusive setting I rather we are explicit when enabling it. But I do agree having the lifecycle ignore its a good additional settings given that we are not looking to change the behavior once set

Reply:
Ok, if we legitimately want the default to be off that's fine. I thought we were sacrificing a prior decision about defaults for compatibility.